charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -1,5 +1,8 @@
 {{- $datacenter := (index .Values.cassandra.datacenters 0) -}}
-{{- $medusaImage := (printf "%s:%s" .Values.backupRestore.medusa.image.repository .Values.backupRestore.medusa.image.tag) -}}
+{{- $medusaImage := (printf "%s:%s" .Values.medusa.image.repository .Values.medusa.image.tag) -}}
+{{- $cassandrUid := 999 -}}
+{{- $cassandraGid := 999 -}}
+{{- $medusaUid := 999 -}}
```

Comment:
This is kinda useless since medusa uses username cassandra

Reply:
👍